lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -86,7 +86,7 @@ local function maketype(typeid)
 	return typename or "unknown"
```

Comment:
You're right, this can never be hit - if `tp[1]` is nil then `tp[1]:lower()` - the only way this could happen is if `tp[1]:lower()` runs successfully and returns `nil`.